lnrpc/rpc.proto

```diff
@@ -1517,6 +1517,9 @@ message GetInfoResponse {
 
     // Whether we consider ourselves synced with the public channel graph.
     bool synced_to_graph = 18 [json_name = "synced_to_graph"];
+
+    /// Features we advertised in our init message.
```

Comment:
nit: this is the union of all features advertised in init messages, node announcments, and invoices